api_app/serializers/job.py

```diff
@@ -422,7 +423,23 @@ class JobTreeSerializer(ModelSerializer):
 
     class Meta:
         model = Job
-        fields = ["pk", "pivot_config"]
+        fields = [
+            "pk",
+            "pivot_config",
+            "observable_name",
+            "file_name",
+            "is_sample",
```

Comment:
Supposition (maybe i'm wrong, Idk): these 3 fields will be used in the frontend to show data about job like: `if is_sample ? file_name : observable-name`. In the backend (class Job) there is a cached property that does this job, Can we return it instead of the fields and perform the same logic in the frontend ?